src/test/java/com/zoftko/felf/services/FelfServiceTests.java

```diff
@@ -0,0 +1,56 @@
+package com.zoftko.felf.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.zoftko.felf.dao.InstallationRepository;
+import com.zoftko.felf.dao.ProjectRepository;
+import com.zoftko.felf.entities.Installation;
+import com.zoftko.felf.entities.Project;
+import com.zoftko.felf.models.RepositoryInstallation;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import reactor.core.publisher.Mono;
+
+@SpringBootTest(classes = FelfService.class)
+class FelfServiceTests {
+
+    @MockBean
+    GithubService githubService;
+
+    @MockBean
+    InstallationRepository installationRepository;
+
+    @MockBean
+    ProjectRepository projectRepository;
+
+    @Autowired
+    FelfService felfService;
+
+    @Test
+    void getProjectDataExists() {
+        var owner = "crazybolillo";
+        var repo = "felf";
+        var install = 1234;
+        var sender = 6789;
+
+        when(githubService.getRepositoryInstallation(owner, repo))
+            .thenReturn(
+                Mono.just(new RepositoryInstallation(install, "main", Map.of("pull_request", "write"), null))
+            );
+
+        var mockedInstall = new Installation();
+        mockedInstall.setId(install);
+        mockedInstall.setSender(sender);
+        when(installationRepository.findByAccountLogin(owner)).thenReturn(Optional.of(mockedInstall));
+
+        when(projectRepository.findByFullName("crazybolillo/felf")).thenReturn(Optional.of(new Project()));
+
+        var projectData = felfService.getProjectData(owner, repo);
+        assertThat(projectData.hasPermissions()).isTrue();
```

Comment:
This should and is probably already tested somewhere else, since the service does not concern itself with how projectData behaves.